README.md

```diff
@@ -458,28 +458,20 @@ module.exports = {
 
 ### Customizing babel config
 
-In order to extend our usage of `babel`, you can define a function that extends its config via `next.config.js`.
-
-The following example config shows you how to use `babel-preset-stage-0` with your app.
+In order to extend our usage of `babel`, you can simply define a `.babelrc` file in the root of your app.
+If found, we'll use it. Here's an example `.babelrc` file:
 
 ```js
-// This file is not going through babel transformation.
-// So, we write it in vanilla JS
-// (But you could use ES2015 features supported by your Node.js version)
-
-module.exports = {
-  // config is the set of options we pass to our babel-loaders's query option
-  babel: function (config, { dev }) {
-    // Add the stage-0 preset.
-    // Make sure to use 'require.resolve' otherwise we won't be able to find it.
-    config.presets.push(require.resolve('babel-preset-stage-0'))
-
-    // Important: return the modified config
-    return config
-  }
+{
+  "presets": [
+    "next/babel",
+    "stage-0"
+  ],
 }
 ```
 
+Although it's not required, you will need to add `next/babel` as a preset. That's the default preset we use for all Next.js apps.
```

Comment:
so, we removed support for `babel` in `next.config.js` right? Otherwise, we also have to make a mention of that here